pkg/framework/context/context.go

```diff
@@ -27,6 +28,7 @@ type Provider struct {
 	storeProvider            storage.Provider
 	transientStoreProvider   storage.Provider
 	kms                      kms.KMS
```

Comment:
Seems like this field should be called crypto to match the rest.

Reply:
sure will rename to crypto..

Reply:
done